cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -298,6 +299,59 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
         }
     });
 
+    SubsectionShareLinkXBlockModal = CourseOutlineXBlockModal.extend({
+
+        initializeEditors: function() {
+            var tabsTemplate;
+            var tabs = this.options.tabs;
+            if (tabs && tabs.length > 0) {
+                    tabsTemplate = this.loadTemplate('subsection-share-link-modal-tabs');
+                    HtmlUtils.setHtml(this.$('.modal-section'), HtmlUtils.HTML(tabsTemplate({tabs: tabs})));
+                    _.each(this.options.tabs, function(tab) {
+                        // eslint-disable-next-line prefer-spread
+                        this.options.editors.push.apply(
+                            this.options.editors,
+                            _.map(tab.editors, function(Editor) {
+                                return new Editor({
+                                    parent: this,
+                                    parentElement: this.$(`.modal-section .${tab.name}`),
+                                    model: this.model,
+                                    xblockType: this.options.xblockType,
+                                });
+                            }, this)
+                        );
+                    }, this);
```

Comment:
I would use this.options in a variable
```suggestion
          let options = this.options;
          _.each(options.tabs, function(tab) {
              // eslint-disable-next-line prefer-spread
              options.editors.push.apply(
                  options.editors,
                  _.map(tab.editors, function(Editor) {
                      return new Editor({
                          parent: this,
                          parentElement: this.$(`.modal-section .${tab.name}`),
                          model: this.model,
                          xblockType: options.xblockType,
                          enable_proctored_exams: options.enable_proctored_exams,
                          enable_timed_exams: options.enable_timed_exams
                      });
                  }, this)
              );
          }, this);
```

Reply:
Done